docs/pages/accounts/audit-logs.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Audit logs
+description: A list of changes that happened to selected entities owned by an account.
+---
+
+## What are audit logs?
+
+Our implementation of audit logs will record actions made with Expo services by accounts. Recorded data will include what object was affected, what type of modification was made, who performed the action, and when this activity occurred.
+
+Because audit logs can only be created and never modified or deleted, they will serve as a source of truth to help monitor events and possibly debug issues occurring within accounts and products.
```

Comment:
```suggestion
Since audit logs can only be created and never modified or deleted, they serve as a source of truth to help monitor events and debug issues occurring within accounts.
```
- Let's make this a bit more concise.
- I removed "products" since that doesn't add to any info. If we can describe it or explain it, that will be helpful.